internal/slicer/slicer_test.go

```diff
@@ -1051,6 +1051,42 @@ var slicerTests = []slicerTest{{
 						content.list("/foo-bar/")
 		`,
 	},
+}, {
+	summary: "Valid same hard link in two slices in the same package",
```

Comment:
```suggestion
	summary: "Valid hard link in two slices in the same package",
```

Reply:
Updated in [4313070](https://github.com/letFunny/chisel/pull/17/commits/4313070b5a0af67b4c1cdca323c0eb03d7f882b9).